docs/pages/application-access/getting-started.mdx

```diff
@@ -67,45 +84,60 @@ $ tctl tokens add \
     --app-uri=http://localhost:3000
 ```
 
-<Tabs>
-<TabItem label="Docker">
+## Step 2/5. Start Grafana
 
-## Step 2/4. Start Grafana
+You can start Grafana in a Docker container or Kubernetes cluster by running the appropriate steps.
+For more information, click **Docker** or **Kubernetes cluster**.
 
-We've picked Grafana for this tutorial since it's very easy to run with zero
-configuration required. If you have another web application you'd like to
-expose, skip over to **Step 3**.
+<Tabs>
+<TabItem label="Docker">
 
-Grafana can be launched in a Docker container with a single command:
+You can start Grafana in a Docker container by running the following command:
 
 ```code
 $ docker run -d -p 3000:3000 grafana/grafana
```

Comment:
FYI The merge conflict that will need to be resolved is likely on this line, where we recently fixed this to provide a name for the container.

Reply:
Yep, I saw that go in.